tests/e2e/packages/verify_packages_install_deb.sh

```diff
@@ -44,7 +44,7 @@ CI_DIR=/tmp/k8s-ci
 mkdir -p $CI_DIR
 mkdir -p /opt/cni/bin
 CI_VERSION=$(curl -sSL https://dl.k8s.io/ci/latest.txt)
-CI_URL="gs://k8s-release-dev/ci/$CI_VERSION-bazel/bin/linux/amd64"
+CI_URL="gs://k8s-release-dev/ci/$CI_VERSION/bin/linux/amd64"
```

Comment:
i will log an issue to remove tests/e2e/packages/

these tests are no longer running and instead there are some publish / verify jobs for packages here:
https://k8s-testgrid.appspot.com/sig-release-releng-informing